src/main/java/com/to/ordermanagementservice/service/impl/SimpleOrderService.java

```diff
@@ -37,61 +48,23 @@ public List<OrderDetails> getAllOrders() {
 
         List<OrderDetails> list = new ArrayList<>(orders.size());
         for (Order order : orders) {
-            OrderDetails orderDetails = collectOrderDetails(order);
+            OrderDetails orderDetails = orderMapper.toOrderDetails(order);
+            orderDetails.setTotalPrice(getTotalPriceForOrder(orderDetails.getOrderItems()));
+            orderDetails.setOrderItems(collectOrderItemDetails(order.getId()));
```

Comment:
please create `collectOrderDetails` method (as it used to be) and put these 3 lines in that method. we will re-use it in the future.

Reply:
method collectOrderDetails created